.gitlab-ci.yml

```
@@ -34,6 +34,15 @@ build-verify:
     - docker push $APP_DEV_BASE_IMAGE
     - docker push $APP_PROD_IMAGE
 
+rspec:
```

please move this into build job instead so that we won't build prod docker image and push images to the registry when rspec tests fail.

could we get the rspec tests run with app-dev instead of app-prod? we should test as soon as possible to fail fast. By this, we got faster feedback and less time for CI job when there are failed tests.

---

do you mean app-dev-base? rspec need a test database to run but currently this service doesnt have it, or shall I add a db link to it?

---

I mean why not "app-dev"? why do you need "app-prod" instead?